src/common/interfaces/event.interface.ts

```diff
@@ -113,7 +113,7 @@ export interface DsyncGroupUserAddedEvent extends EventBase {
   data: {
     directoryId: string;
     user: DirectoryUser;
-    group: Pick<EventDirectoryGroup, 'id' | 'name'>;
+    group: EventDirectoryGroupResponse;
```

Comment:
This should be an `EventDirectoryGroup`

Reply:
Yep, good call - initially these were both throwing errors with EventDirectoryGroupResponse, but I just needed to adjust the event.serializer.ts file a bit to make it work.